internal/services/dynatrace/helper.go

```diff
@@ -0,0 +1,181 @@
+package dynatrace
```

Comment:
Is there a specific reason to put these in a separate file? Otherwise, can we put them back to the resource file, also, define the schemas in the root schema composite literally?